castle/cms/interfaces/controlpanel.py

```diff
@@ -503,3 +504,6 @@ class ICrawlerConfiguration(Interface):
         description=u'User agent to use when crawling sites',
         default=u'CastleCMS Crawler 1.0'
     )
+
+
+
```

Comment:
Is there a commit hook tool that recommended these whitespace changes? If not, please revert them.